robust_pdf_plot.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    _, _ = plot_outlier_effect(save_name="robust_pdf_plot_outliers_latexified.pdf", outliers=[8, 8.75, 9.5])
</pre><p>Ideally, above two plots should have same  y_range to actually know the outlier effect on gaussian distribution. (for example check book figure they have same range 0 to 0.5 for both figures) If they have different height then it is visually difficult to see that gaussian height is decreased.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done, axis are static now with range [0.0,0.5]</p>